include/RED4ext/CommandLine-inl.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#ifdef RED4EXT_STATIC_LIB
+#include <RED4ext/CommandLine.hpp>
+#endif
+
+#include <RED4ext/Detail/AddressHashes.hpp>
+#include <RED4ext/Relocation.hpp>
+#include <RED4ext/StringView.hpp>
+
+RED4EXT_INLINE RED4ext::CommandLine* RED4ext::CommandLine::Get() noexcept
+{
+    // NOTE: GetAddr is actually necessary here
+
+    static CommandLine* ptr{
+        RED4ext::UniversalRelocPtr<RED4ext::CommandLine>(RED4ext::Detail::AddressHashes::CommandLine).GetAddr()};
+    return ptr;
+}
+
+RED4EXT_INLINE bool RED4ext::CommandLine::HasLaunchParameter(RED4ext::StringView aParam) noexcept
+{
+    using func_t =
```

Comment:
you dont have to specify params, makes it shorter and imo more readable (as you only care about types of arguments)